kotlin-source/src/main/kotlin/com/example/flow/ExampleFlow.kt

```diff
@@ -65,88 +69,41 @@ object ExampleFlow {
             progressTracker.currentStep = GENERATING_TRANSACTION
             // Generate an unsigned transaction.
             val txCommand = Command(IOUContract.Commands.Create(), iou.participants.map { it.owningKey })
-            val unsignedTx = TransactionType.General.Builder(notary).withItems(iou, txCommand)
+            val txBuilder = TransactionType.General.Builder(notary).withItems(iou, txCommand)
 
             // Stage 2.
             progressTracker.currentStep = VERIFYING_TRANSACTION
             // Verify that the transaction is valid.
-            unsignedTx.toWireTransaction().toLedgerTransaction(serviceHub).verify()
+            txBuilder.toWireTransaction().toLedgerTransaction(serviceHub).verify()
 
             // Stage 3.
             progressTracker.currentStep = SIGNING_TRANSACTION
-            val partSignedTx = serviceHub.signInitialTransaction(unsignedTx)
+            // Sign the transaction.
+            val partSignedTx = serviceHub.signInitialTransaction(txBuilder)
 
             // Stage 4.
-            progressTracker.currentStep = SENDING_TRANSACTION
-            // Send the state across the wire to the designated counterparty.
-            // -----------------------
-            // Flow jumps to Acceptor.
-            // -----------------------
-            send(otherParty, partSignedTx)
+            progressTracker.currentStep = GATHERING_SIGS
+            // Send the state to the counterparty, and receive it back with their signature.
+            val fullySignedTx = subFlow(CollectSignaturesFlow(partSignedTx))
```

Comment:
you can add the param, `GATHERING_SIGS.childProgressTracker()`, likewise for Finalityflow.